pom.xml

```diff
@@ -42,7 +42,7 @@
 
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<java.version>1.6</java.version>
+		<java.version>1.8</java.version>
```

Comment:
Feel free to double check that everything compiles, and that the output library has the correct version. Maybe try adding a Java 8-only dependency to core, or add some Java 7 code, and see if the build fails as it should.